src/style.css

```diff
@@ -1 +1,10 @@
 /* styles go here */
+fieldset {
+  margin: 20px;
```

Comment:
now it applies to all margins, but you need only vertical. and don't set both top and bottom, chose only one